x-pack/platform/test/fleet_api_integration/apis/fleet_telemetry.ts

```diff
@@ -31,17 +31,6 @@ export default function (providerContext: FtrProviderContext) {
       await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.load('x-pack/platform/test/fixtures/es_archives/fleet/empty_fleet_server');
       await fleetAndAgents.setup();
-    });
-
-    after(async () => {
-      await kibanaServer.savedObjects.cleanStandardList();
-      await esArchiver.unload('x-pack/platform/test/fixtures/es_archives/fleet/empty_fleet_server');
-      if (pkgVersion) {
-        await supertest.delete(`/api/fleet/epm/packages/fleet_server/${pkgVersion}`);
-      }
-    });
-
-    before(async () => {
```

Comment:
`mocha/no-sibling-hooks: Unexpected use of duplicate Mocha `before` hook`

I merged `before` hooks